examples/timer/src/main.rs

```diff
@@ -139,7 +139,7 @@ impl Component for App {
                         { &self.time }
                     </div>
                     <div id="messages">
-                        { for self.messages.iter().map(|message| html! { <p>{ message }</p> }) }
+                        { for self.messages.iter().map(|message| html! { <p>{ *message }</p> }) }
```

Comment:
I'm going to fix that in a separate PR.

Related to https://github.com/yewstack/implicit-clone/pull/37